Add docs update for counter forwarding to oss (#12998)
hashicorp/vault: website/content/api-docs/system/internal-counters.mdx

# `/sys/internal/counters`

> The `/sys/internal/counters` endpoints are used to return data about the number of Tokens and Entities in Vault. The information returned by these endpoints are cluster level values.

Reviewer: ```suggestion
The `/sys/internal/counters` endpoints are used to return data about the number of Tokens and Entities in Vault. They return information for the entire cluster.
```
